packages/deployments/contracts/contracts/core/connext/facets/ProposedOwnableFacet.sol

```diff
@@ -245,12 +254,48 @@ contract ProposedOwnableFacet is BaseConnextFacet, IProposedOwnable {
     _setOwner(s._proposed);
   }
 
-  function pause() public onlyOwner {
+  /**
+   * @notice Assign candidate -> router role
+   */
```

Comment:
there are not enough comments here about what the role does or who is allowed to assign it, and why it exists. also missing natspec

Reply:
missed the comments big time, fixed